pkgs/tools/system/supervise/default.nix

```diff
@@ -1,22 +1,15 @@
-{ stdenv, fetchFromGitHub }:
+{ stdenv, fetchzip }:
 
 stdenv.mkDerivation rec {
 
   name = "supervise-${version}";
-  version = "1.0.0";
+  version = "1.1.0";
 
-  src = fetchFromGitHub {
```

Comment:
What is the reason for switching to fetchzip?

Reply:
This tarball is not the raw source, it's after running autoreconf to generate ./configure scripts, and creating a tarball with "make distcheck"

Reply:
Then you could just add ```autoreconfHook``` to the ```nativeBuildInputs``` and that would take care of that.

Reply:
Yes, I am aware, but this is the official release tarball and it does not need autoreconfHook.

Reply:
I'm honestly not sure if we gain anything by running the autoreconfHook against the raw source vs using the release tarball but the recommended way to fetch from github is using ```fetchFromGitHub``` rather than ```fetchurl```.

Reply:
There's no way to fetch the release tarball using fetchFromGithub. Ignore the fact that the release tarball (which is an opaque artifact which is not automatically derivable from the Git repo) is hosted on Github, the release tarball might as well be from anywhere else.

Reply:
@peterhoeg Ping? I really don't think this is a big issue, this is just something that fetchFromGithub is not supposed to do.

If you want I can host the release tarball on my own website so it doesn't say "github" in the URL :)